components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.html

```diff
@@ -19,7 +19,7 @@ <h2 class="heading-title">{{ desktop.name }}</h2>
       <h3>Check-in History</h3>
       <div class="heading-actions">
         <chef-button secondary (click)="updateCheckInDays()">
-          <span>Last {{ (checkinNumDays === 15 ? 2 : 4) }} Weeks</span>
+          <span>Last {{ (checkInNumDays === 15 ? 2 : 4) }} Weeks</span>
```

Comment:
Hmm... I find a lot of the magic number 15 being spread around the desktop; might be worthwhile to make it a named constant.